test/validation/api/crypto/odp_crypto_test_inp.c

```diff
@@ -1193,7 +1202,9 @@ static void check_alg(odp_crypto_op_t op,
 			continue;
 		}
 
-		alg_test(op, cipher_alg, auth_alg, &ref[idx], digest_offs,
+		alg_test(op, cipher_alg, auth_alg, AUTH_PLAINTEXT, &ref[idx], digest_offs,
+			 is_bit_mode_cipher, is_bit_mode_auth);
+		alg_test(op, cipher_alg, auth_alg, AUTH_CIPHERTEXT, &ref[idx], digest_offs,
```

Comment:
I'd update spec text to say that value of auth_cipher_text is ignored, when operation is auth or crypto only. Tests can leave the value then to default and less combinations to test for those operations.

Reply:
That spec update would not really change anything since auth_cipher_text does not anyway have any effect when one of the algorithms is null. So saying that it is ignored does not change the semantics of the API at all. It also does not change the fact that regardless of what the API says, an implementation may have a bug that it only works if auth_cipher_text is set in a certain way, in which case testing both values reveals the problem.